server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -212,13 +212,14 @@ main {
   .cf-program-card-group {
     display: grid;
     grid-template-columns: 1fr 1fr;
+    margin-left: -0.5rem;
+    margin-right: -0.5rem;
   }
 }
 
 // Desktop: Program card grid has 3 columns
 @media (min-width: 64em) {
   .cf-program-card-group {
-    display: grid;
```

Comment:
**question (non-blocking):** Why remove grid on desktop?

Reply:
I realized `grid` in the `min-width:40em` rule also applies to desktop, so it's redundant.

Reply:
good catch!